Snakefile

```diff
@@ -201,6 +207,20 @@ rule add_electricity:
         "scripts/add_electricity.py"
 
 
+if config["enable"].get("download_osm_data", True):
+
+    rule download_osm_data:
+        params:
+            countries=config["countries"],
+        output:
+            "resources", RDIR, "osm", "raw"
```

Comment:
this may be something like: "resources/" + RDIR + "/osm/raw/all_buildings.geojson"